index.html

```diff
@@ -0,0 +1,578 @@
+
+## `@core(export: Boolean)` *on any [`TypeSystemDirectiveLocation`](https://spec.graphql.org/June2018/#DirectiveLocations)*
+
+`@core(export:)` can occur at any location. Elements with `@core(export: true)` will always be included in the export schema. Elements with `@core(export: false)` will always be excluded from the export schema.
+
+## Requiring core schemas within a spec
+
+Specifications **may** include the following language to instruct schema authors to use core schema mechanisms to reference them:
+
+> SPEC_NAME schema documents **must** be valid [core schemas](/core/drafts/pre-0), and must
+> reference SPEC_NAME and the specification URL `SPEC_URL`:
+>
+> ```graphql
+> schema
+>   @core(using: "https://lib.apollo.dev/core/v0.1")
+>   @core(using: SPEC_URL)
+> {
+>   query: Query
+> }
+> ```
+
+`SPEC_NAME` and `SPEC_URL` should be replaced with the names and URLs for the specification. For example:
+
+> Example Spec schema documents **must** have a `schema` declaration with a [`@core` directive](/core/drafts/pre-0)
+> and the specification URL `https://specs.example.com/example/v1.0`:
+>
+> ```graphql
+> schema
+>   @core(using: "https://lib.apollo.dev/core/v0.1")
+>   @core(using: "https://specs.example.com/example/v1")
+> {
+>   query: Query
+> }
+> ```
+
+## Spec URLs
+
+Spec URLs serve two main purposes:
+  - Directing human readers to documentation about the specification(s) in use.
+  - Providing tools with information about the specs in use, along with enough information to select and invoke an implementation
+  
+Spec URLs **should** be [RFC 3986 URLs](https://tools.ietf.org/html/rfc3986). When viewed, the URL **should** provide the content of the selected version of the specification in some human-readable form; a human reader should be able to click the link and go to the correct version of the docs.
+
+Although they are not prohibited from doing so, it's assumed that processors will not load the content of spec URLs. Published specifications are not required to be machine-readable, and [this spec](.) places no requirements on the structure or syntax of the content to be found there.
+
+There are, however, requirements on the structure of the URL itself:
+
+<figure>
+  <figcaption>
+    <a name=basic-anatomy-of-a-spec-url href=#basic-anatomy-of-a-spec-url class=a-header>Basic anatomy of a spec URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://spec.example.com/a/b/c/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) **should** contain the spec's [default prefix](#prefix) and a [version tag](#versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the spec's *identity*. For the above example,
```

Comment:
You use the phrases `default prefix`, `name`, and `default name` in various places in the spec.  Can we choose just one? Or I can understand why you might use `name` in some places and `default prefix` in others maybe. Don't understand `default name` though.